products/sds/playbooks/oioswift.yml

```diff
@@ -13,6 +13,12 @@
   environment: "{{ openio_environment }}"
 
   roles:
+    - role: memcached
+      openio_memcached_namespace: "{{ namespace }}"
+      openio_memcached_gridinit_dir: "/etc/gridinit.d/"
+      openio_memcached_gridinit_file_prefix: "{{ namespace }}-"
+      when: groups['keystone'] | length == 0
```

Comment:
Maybe cleaner to have a var like `openio_with_keystone: yes` instead of repeating `groups['keystone'] `